pangeo_forge_recipes/executors/base.py

```diff
@@ -27,12 +27,27 @@ class Stage:
     mappable: Optional[Iterable] = None
     annotations: Optional[StageAnnotations] = None
 
+    @property
+    def ismappable(self):
+        return True if self.mappable is not None else False
```

Comment:
I feel like this property is not necessary because `if stage.mappable` will accomplish the same thing.